### PR TITLE
Added Loading experience for Services and Associated Pods in Services…

### PR DESCRIPTION
**Pull request by dabasnidhi**
test-pod- http://nidabas-desk/DefaultCollection/p1/_serviceEndpoints/172104eb-38d7-40b7-88ed-7e95d242c073/kubernetesSummary?namespace=default
Services Loading- 
image
Pods loading-
image

Workloads Landing is fetched by data provider, hence loading not required there.
Also show Images Layers information sorted by Created Date-
image

### CODE REVIEW

File: src/WebUI/Pods/PodsDetails.tsx [hunk 1]
```diff
@@ -36,6 +37,7 @@ export interface IPodsDetailsState {
     pods?: V1Pod[];
     parentName?: string;
     parentKind?: string;
+    podsLoading?: boolean;
```
Comment (madhuv-msft) [inline src/WebUI/Pods/PodsDetails.tsx]: isLoading? is a better name?

File: src/WebUI/Pods/PodsDetails.tsx [hunk 2]
```diff
@@ -178,65 +188,67 @@ export class PodsDetails extends BaseComponent<IPodsDetailsProperties, IPodsDeta
...
-                        ? (<Splitter
-                            fixedElement={SplitterElementPosition.Near}
-                            initialFixedSize={this._initialFixedSize}
-                            minFixedSize={this._initialFixedSize}
-                            onRenderFarElement={() => rightPanel}
-                            onRenderNearElement={() => leftPanel}
-                            nearElementClassName="pods-details-left-pane"
-                        />)
+            {
+                leftPanel
```
Comment (madhuv-msft) [inline src/WebUI/Pods/PodsDetails.tsx]: formatting issue?

File: src/WebUI/Services/ServicesPivot.tsx [hunk 3]
```diff
@@ -20,9 +21,12 @@ import { IVssComponentProperties } from "../Types";
 import { ServicesActionsCreator } from "./ServicesActionsCreator";
 import { ServicesFilterBar } from "./ServicesFilterBar";
 import { ServicesStore } from "./ServicesStore";
+import * as Resources from "../Resources";
+import "./ServicesPivot.scss";
 
 export interface IServicesPivotState {
     serviceList?: V1ServiceList;
+    servicesLoading?: boolean;
```
Comment (madhuv-msft) [inline src/WebUI/Services/ServicesPivot.tsx]: isLoading?

File: src/WebUI/Services/ServicesPivot.tsx [hunk 4]
```diff
@@ -38,21 +42,27 @@ export class ServicesPivot extends BaseComponent<IServicesPivotProps, IServicesP
...
+            return <Spinner className={"flex flex-grow loading-services"} size={SpinnerSize.large} label={Resources.LoadingServicesSpinnerLabel} />;
+        }
+
         return (
             <>
-                {this._getFilterBar()}
-                <div className={css("services-pivot-data", "k8s-pivot-data")}>
-                    {this._getContent()}
-                </div>
+            { this._getFilterBar() }
```
Comment (madhuv-msft) [inline src/WebUI/Services/ServicesPivot.tsx]: formatting

File: src/WebUI/WorkLoads/WorkloadDetails.tsx [hunk 5]
```diff
@@ -58,6 +59,7 @@ export interface IWorkloadDetailsState {
     showSelectedPod: boolean;
     showImageDetails: boolean;
     selectedImageDetails: IImageDetails | undefined;
+    podsLoading?: boolean;
```
Comment (madhuv-msft) [inline src/WebUI/WorkLoads/WorkloadDetails.tsx]: isPodsLoading?

File: src/WebUI/Resources.js [hunk 6]
```diff
@@ -92,4 +92,6 @@ define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], fu
     exports.SizeText = "Size";
     exports.SummaryHeaderSubTextFormat = "{0} (cluster)";
     exports.TagsText = "Tags";
+    exports.LoadingPodsSpinnerLabel = "Loading pods...";
```
Comment (madhuv-msft) [inline src/WebUI/Resources.js]: As suggested by PM, use 'Loading associated pods...'

File: src/WebUI/Services/ServiceDetails.tsx [hunk 7]
```diff
@@ -44,6 +46,7 @@ export interface IServiceDetailsProperties extends IVssComponentProperties {
 export interface IServiceDetailsState {
     service: IServiceItem | undefined;
     pods: Array<V1Pod>;
+    podsLoading?: boolean;
```
Comment (madhuv-msft) [inline src/WebUI/Services/ServiceDetails.tsx]: isPodsLoading?

Comment (dabasnidhi) [reply src/WebUI/Services/ServiceDetails.tsx]: Changed to arePodsLoading

File: src/WebUI/WorkLoads/WorkloadDetails.tsx (hunk 5 above)
Comment (dabasnidhi) [reply src/WebUI/WorkLoads/WorkloadDetails.tsx]: Changed to arePodsLoading

File: src/WebUI/Services/ServicesPivot.tsx (hunk 3 above)
Comment (dabasnidhi) [reply src/WebUI/Services/ServicesPivot.tsx]: Changed to isLoading

File: src/WebUI/Pods/PodsDetails.tsx (hunk 1 above)
Comment (dabasnidhi) [reply src/WebUI/Pods/PodsDetails.tsx]: Changed to isLoading

**Pull request merged**